pkg/destroy/powervs/cloudconnection.go

```diff
@@ -42,22 +42,28 @@ func (o *ClusterUninstaller) listCloudConnections() (cloudResources, error) {
 	default:
 	}
 
+	result := []cloudResource{}
+
 	cloudConnections, err = o.cloudConnectionClient.GetAll()
 	if err != nil {
-		return nil, errors.Wrapf(err, "failed to list cloud connections")
+		message := err.Error()
+		o.Logger.Debugf("HAMZY: %s", message)
```

Comment:
Maybe a debug message without your name if you intend on keeping this?

Reply:
Sigh.  Personal debugging when writing the code :(